go/vt/vitessdriver/driver_go18.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.8
+
+// TODO(sougou): Merge this with driver.go once go1.7 is deprecated.
```

Comment:
Also, original non-Context methods can have an implementation `panic("See Context method")` once 1.8 is the minimum.